lib/NfcoreSchema.groovy

```diff
@@ -345,11 +349,20 @@ class NfcoreSchema {
             else {
                 def param = ex_json['pointerToViolation'] - ~/^#\//
                 def param_val = params_json[param].toString()
-                log.error "* --${param}: ${ex_json['message']} (${param_val})"
+                if (enums.containsKey(param)) {
```

Comment:
Hmm, we should stick to the template files strictly for these as they will receive updates from time to time and making such changes potentially causes more trouble than it helps. 

Would also be good to merge in the template first and then applying your other changes on top of it?

Reply:
I agree. I was shuffling with different nf-core versions and the result is not ideal. 
Applying #195 first is a good strategy I think

Reply:
star/genomegenerate is a nf-core module for example, I think we can take existing modules from nf-core, but keep what is local as local for now